src/NzbDrone.Core/Datastore/Migration/0148_add_on_delete_to_notifications.cs

```diff
@@ -0,0 +1,14 @@
+﻿using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(148)]
+    public class add_on_delete_to_notifications : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Notifications").AddColumn("OnDelete").AsBoolean().WithDefaultValue(false);
+        }
+    }
+}
```

Comment:
Nit: new line at end of file

Reply:
@markus101  any idea why git/github isn't respecting the line at the end of my file? 
![image](https://user-images.githubusercontent.com/19610103/102621179-2124d200-4137-11eb-88dc-05bbce84b368.png)